py_gtfs_rt_ingestion/py_gtfs_rt_ingestion/convert_gtfs_rt.py

```diff
@@ -130,7 +135,14 @@ def gz_to_pyarrow(self, filename: str) -> Optional[str]:
 
             # parse timestamp info out of the header
             feed_timestamp = json_data["header"]["timestamp"]
-            timestamp = datetime.utcfromtimestamp(feed_timestamp)
+            timestamp = datetime.fromtimestamp(feed_timestamp, timezone.utc)
+
+            # skip files that have been generated after the start of the current
+            # hour. don't add them to archive or error so that they are picked
+            # up next go around.
+            if timestamp > self.start_of_hour:
```

Comment:
```suggestion
            if timestamp >= self.start_of_hour:
```

Don't want to process files with timestamps equal to the hour.